### PR TITLE
DataType.yaml: Add precision attribute for floating points

### PR DESCRIPTION
**Pull request by siddhpant**
Add precision attribute for data types like float, double, etc.

Updated #44 and #46 regarding this (_I had earlier rather erroneously used the attribute without adding it in this file_).

### CODE REVIEW

File: models/DataType.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 keyword: String  # Keyword that identifies the type i.e. int, str
 # Integer attributes
-integer-min: String  # String representing minium integer or uncertain
+integer-min: String  # String representing minimum integer or uncertain
 integer-max: String  # String represetinng maximum integer or uncertain
 integer-signed: Boolean  # Whether the datatype is signed integer or not
 delimiters: Delimiter[] # If the datatype has any delimiters
 literals: Keyword[] # List of literals for a datatype
+precision: Integer # Integer representing precision of floating point data types.
```
Comment (jayvdb) [inline models/DataType.yaml]: two spaces before `#`

(This repo has a .coafile now)

**Comment by jayvdb** [comment]
And you need to do a manual rebase.  Read https://coala.io/git and https://coala.io/newcomer , and ask for help if you need it.  Dont just run random commands or click silly GitHub green buttons to hope it works.  Dont close the PR to create another one -- we'll close your new PR and re-open the old one.  Learning `git` is a mandatory part of participation in coala org.

File: models/DataType.yaml (hunk 1 above)
Comment (siddhpant) [reply models/DataType.yaml]: Done.

**Comment by siddhpant** [comment]
Rebased.

File: models/DataType.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 keyword: String  # Keyword that identifies the type i.e. int, str
 # Integer attributes
-integer-min: String  # String representing minium integer or uncertain
+integer-min: String  # String representing minimum integer or uncertain
 integer-max: String  # String represetinng maximum integer or uncertain
 integer-signed: Boolean  # Whether the datatype is signed integer or not
 delimiters: Delimiter[]  # If the datatype has any delimiters
 literals: Keyword[]  # List of literals for a datatype
+precision: Integer  # Integer representing precision of floating point data types.
```
Comment (jayvdb) [inline models/DataType.yaml]: line too long (82 > 80 characters) (line-length)

*Origin: YAMLLintBear, Section: `yaml`.*

Comment (siddhpant) [reply models/DataType.yaml]: Okay, please check now!

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/48/commits/d8342be795f6a401a39b6ab1aeee6374d577c475

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**